dockerfiles/vcpkg/ubuntu-20.04.dockerfile

```diff
@@ -1,9 +1,18 @@
 ARG VCPKG_REF=latest
-FROM hpccbuilds/vcpkg-ubuntu-20.04:$VCPKG_REF
+ARG DOCKER_NAMESPACE=hpccbuilds
+FROM ${DOCKER_NAMESPACE}/vcpkg-ubuntu-20.04:$VCPKG_REF
```

Comment:
This will change shortly (you may need to change it depending on which PR get pulled first)